src/encoded/authentication.py

```diff
@@ -279,19 +340,41 @@ def logout(context, request):
     The front-end handles logging out by discarding the locally-held JWT from
     browser cookies and re-requesting the current 4DN URL.
     """
-    #request.session.invalidate()
-    #request.response.headerlist.extend(forget(request))
+    request_parts = urlparse(request.referrer)
+    request_domain = request_parts.hostname
+
+    # Deletes the cookie
+    request.response.set_cookie(
+        name='jwtToken',
+        value=None,
+        domain=request_domain,
+        max_age=0,
+        path='/',
+        overwrite=True
+    )
```

Comment:
Does this not need to specify the http_only and secure options in order to be able to find the cookie it wants to delete? or does it not matter?

Reply:
It shouldn't matter I don't think, at least testing showed it to work. As long as the request is still to an https endpoint, then it will unset the https.

One place it didn't work is because I set my localhost for CGAP to https, and 4DN to be http. Cookie was shared between both instances, but http://localhost couldn't delete a cookie created by https://localhost. Which I think is to be expected.